### ISSUE TITLE
Support wallets on Cardano mainnet

### ISSUE THREAD

**Issue by aspiers** [rotki/rotki#2488]
We have seen an almost meteoric rise of Cardano's popularity recently.  At the time of writing it's [the 3rd biggest blockchain project by marketcap](https://www.coingecko.com/en/coins/cardano), eclipsing BNB, USDT, and DOT.  So I think it's worth having a GitHub issue as a placeholder for discussion around adding support for wallets on Cardano's mainnet.  ICBW but I expect this will probably become a more common request from Rotki users over the next few months.

BTW other portfolio trackers such as CoinStats, Koinly, and Accointing all support Cardano wallets already.

**Comment by aspiers** [comment]
It's worth noting that as of the "Mary" protocol update on 1st March 2021, [Cardano now supports multiple assets](https://www.coinspeaker.com/cardano-multi-asset-blockchain-mary/) so in the future there will most likely be a demand for portfolio trackers to track not just ADA but other native on-chain tokens too.  But maybe it's better to track that in a separate issue to avoid this one becoming too bloated.

**Comment by sampsojp** [comment]
Please make this happen. Thank you.

**Comment by sbellem** [comment]
I'd be happy to help with this, on a part time basis.

**Comment by LefterisJP** [comment]
Hey @sbellem thanks! We have a developer chat in discord where we can coordinate: https://discord.gg/aGCxHG7

**Comment by Faustino-CIMA** [comment]
Yes, please add full Cardano support. Thank you.

**Comment by cybern0id** [comment]
I've been trying to drum up some support in the Cardano Technical Community discord: https://discord.com/channels/826816523368005654/862682382314635274/915744143081365524

Maybe Rotki devs might like to submit a proposal to Cardano's project Catalyst where winning proposals are awarded funding to facilitate related projects? Though I know you guys are stretched.

I can't code so I'm trying to contribute in the best way I can right now unless someone has other suggestions?

Having fully functioning tax reporting for Cardano must be on many people's wish list, surely? Or am I the only one? Not many seem to be talking about crypto taxes in particular.

Instant python-expert brain plug-in module please Mr Musk....  :smile:

**Comment by mvdkleijn** [comment]
Is anyone actually working on this? I quite like Rotki, but since most of my holdings are in ADA....

**Comment by Faustino-CIMA** [comment]
> Is anyone actually working on this? I quite like Rotki, but since most of my holdings are in ADA....

No, it's sad, I check the Milestones in this repository and there are no mentions about Cardano whatsoever, I checked all the labels up to 1.29.0 .... :-) 

I have a Premium subscription for a year but I will not renew it until there is full support for Cardano.

**Comment by LefterisJP** [comment]
> > Is anyone actually working on this? I quite like Rotki, but since most of my holdings are in ADA....
> 
> No, it's sad, I check the Milestones in this repository and there are no mentions about Cardano whatsoever, I checked all the labels up to 1.29.0 .... :-)
> 
> I have a Premium subscription for a year but I will not renew it until there is full support for Cardano.

Hello @mvdkleijn and @Faustino-CIMA. Please use issues for a simple +1 or "I need this feature too" so we can gauge interest on individual features. Or to discuss technical implementation and UI/UX. For other kind of discussions feel free to join our discord: https://discord.com/invite/aGCxHG7 

As for the when this is going to be implemented we can't give any ETAs. If you check our issues there is a huge backlog of stuff to do for our small team and we prioritize depending on popularity. This is opensource software. The best thing you can do is what Avalance did. They found people who came in and actually developed the avalanche module. That would help us immensely. @cybern0id seems to have started something but I don't see anyone wanting to come and make a PR yet.

**Comment by kieransimkin** [comment]
Cardano native assets are by far the best designed out of all the blockchains I've seen, well worth supporting them :)

**Comment by matiwinnetou** [comment]
+1

**Comment by eehrhardt** [comment]
+1

**Comment by papacarp** [comment]
+1

**Comment by Faustino-CIMA** [comment]
+1

**Comment by jimsjoh** [comment]
+1

**Comment by profd2004** [comment]
if there are talents to do the work, Cardano will be happy to [pay for it](https://projectcatalyst.io/)!

**Comment by Faustino-CIMA** [comment]
+1

**Comment by mvdkleijn** [comment]
**To everyone leaving a +1 comment:** please don't. It's common practice to express your desire for an issue by leaving a thumbs up at the **top** of the issue, on _the issue's description_.
